### ISSUE TITLE
[SCREENREADER, COGNITIVE]: Content for each disability lacks context when read by screenreader

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3682]
## Description

**[SCREENREADER]: Content for each disability lacks context when read by screenreader**

When the screen reader reads the content for each of the disabilities there is nothing indicating what each of the items are. It would help the screen reader experience to have semantic context (such as a table or list items) with labels associated for each data point. The visual presentation of data within a grey box helps to convey the context of the data for sighted users. Screenreader users have no context for the data as it is read out.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to hear context of each piece of data presented so I may understand the context presented.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: https://staging.va.gov/disability/view-disability-rating/rating/

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/view-disability-rating/rating/` in browser
2. Start a screenreading device
3. Hear the data read out for each of the disabilities
4. Verify it's a string of data points without context

### Current Code
```html
<div class="vads-l-col--12 vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2">
    <div class="vads-l-row">
        <div class="vads-l-col--9">
            <p class="vads-u-font-weight--bold vads-u-margin-top--0p25 vads-u-margin-bottom--0 vads-u-margin-x--0 vads-u-font-size--base">
                Allergies due to Hearing Loss
            </p>
        </div>
        <div class="vads-l-col--3">
            <p class="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
                10%
            </p>
        </div>
    </div>
    <div class="vads-l-row vads-u-margin-y--0p5">
        <div class="vads-l-col--12 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
            <p class="vads-u-margin--0">
                <i class="fas fa-medal vads-u-margin-right--0p5"></i>
                <span class="vads-u-font-weight--bold">
                    Service Connected
                </span>
            </p>
        </div>
    </div>
    <div class="vads-l-row">
        <div class="vads-l-col--12 medium-screen:vads-l-col--12">
            <p class="vads-u-margin--0">
                Effective date: 01/05/2012
            </p>
        </div>
    </div>
</div>
```

### Potential solution
Consider a semantic structure that provides context, with labels for each of the data points. It is possible to suppress the labels for screen display, and only have them available for screen readers, however sighted users are likely to benefit from the additional context, so they don't have to figure it out.
```html
<dl>
    <dt>Allergies due to Hearing Loss</dt>
    <dd><dfn>Disability Rating</dfn> 10%</dd>
    <dd><dfn>Origin of Disability</dfn> Service Connected</dd>
    <dd><dfn>Effective Date</dfn> 01/05/2012</dd>
</dl>
```

## WCAG or Vendor Guidance (optional)

Cognitive - The information architecture of the site and design SHOULD be intuitive, allowing the user to easily
find, navigate, read, and interact with the content. 

[WCAG 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html) - Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

**Comment by jason-gcio** [comment]
This was addressed in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4021
